### PR TITLE
[#2912] improve(core): Using class not interface for `Configs`

### PR DESCRIPTION
**Pull request by kiratkumar47**
### What changes were proposed in this pull request?
Changed Configs interface to class since Configs doesn't include any methods but defines a lot of constants.

### Why are the changes needed?
Fix: #2912 

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
manually

### CODE REVIEW

File: core/src/main/java/com/datastrato/gravitino/Configs.java [hunk 1]
```diff
@@ -12,113 +12,120 @@
 import java.util.List;
 import org.apache.commons.lang3.StringUtils;
 
-public interface Configs {
+public class Configs {
```
Comment (jerryshao) [inline core/src/main/java/com/datastrato/gravitino/Configs.java:15]: Hi @kiratkumar47 would you please add a private constructor for this class `Configs`?

Comment (kiratkumar47) [reply core/src/main/java/com/datastrato/gravitino/Configs.java:15]: Hi @jerryshao 
It is done.

**Review by jerryshao — APPROVED** [review]
LGTM. Thanks @kiratkumar47 for your contribution.

**Comment by justinmclean** [comment]
Close as it has been done.

**Pull request merged**